lib/SILGen/SILGenFunction.h

```diff
@@ -664,7 +664,11 @@ class LLVM_LIBRARY_VISIBILITY SILGenFunction
                            SubstitutionMap witnessSubs,
                            IsFreeFunctionWitness_t isFree,
                            bool isSelfConformance);
-  
+
+  SmallVector<ManagedValue, 4>
+  emitKeyPathSubscriptOperands(SubscriptDecl *subscript, SubstitutionMap subs,
```

Comment:
Could you add a brief doc comment here to explain that this function emits arguments for a key path subscript component?